app/views/devise/sessions/new.html.haml

```diff
@@ -1,3 +1,5 @@
+= title "Sign in"
```

Comment:
there are a few other pages like forgot password, resetting the password page (after following forgot password link), resend confirmation email